clusterctl/clusterdeployer/bootstrapclusterprovisioner/externalbootstrapcluster.go

```diff
@@ -37,14 +37,15 @@ func NewExternalCluster(kubeconfigPath string) (*ExternalBootstrapCluster, error
 		return nil, fmt.Errorf("file at %s does not exist", kubeconfigPath)
 	}
 
-	return &ExternalBootstrapCluster{kubeconfigPath:kubeconfigPath}, nil
+	return &ExternalBootstrapCluster{kubeconfigPath: kubeconfigPath}, nil
```

Comment:
Are we leaving this as "ExternalBootstrap" because we also expect to have a different type of FooBoootstrap in the future? Or is that redundant and we should just call this "BootstrapCluster"?

Reply:
I did a bit of re-organization and renaming to help clear it up.